evaluation/assert_quotes_contained_in_chunk.py

```diff
@@ -1,13 +1,102 @@
+import re
+import json
 from typing import Any
+from pprint import pformat, pp
+
+
+def check_quote_in_chunk(quote: str, chunk: str) -> bool:
```

Comment:
Would it be worth using ROUGE here (shouldn't be hard, as it's used elsewhere) - to find if the quote is only slightly modified? Or is it safer to look for alphanumeric matches as the current code does?

Reply:
Yeah, we could. I suppose it depends how strict we want the check to be. Right now, given the actual product uses exact string matches to find timestamps, I think strict makes sense, but I think we should look into doing something fuzzier in the product (and so I guess here too?). Maybe they should even be two separate tests?

This was just a super quick PoC solution though tbh - deffo could do with some refinement